### ISSUE TITLE
Doesn't work with node@10

### ISSUE THREAD

**Issue by zemlanin** [cactorium/node-cint64#2]
Since `node@8`, `cons->NewInstance` was deprecated. In `node@10`, it was removed…

```
$ node -v
v10.6.0

$ npm i

> node-cint64@1.1.3 install /Users/antonverinov/workspace/node-cint64
> node-gyp rebuild

  CXX(target) Release/obj.target/node-cint64/src/cint64.o
In file included from ../src/cint64.cc:3:
../src/int64-type.h:57:26: error: no matching member function for call to 'NewInstance'
        auto ret = cons->NewInstance(argc, argv);
                   ~~~~~~^~~~~~~~~~~
/Users/antonverinov/.node-gyp/10.6.0/include/node/v8.h:3914:44: note: candidate function not viable: requires single argument 'context', but 2 arguments were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
/Users/antonverinov/.node-gyp/10.6.0/include/node/v8.h:3911:44: note: candidate function not viable: requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
In file included from ../src/cint64.cc:3:
../src/int64-type.h:76:26: error: no matching member function for call to 'NewInstance'
        auto ret = cons->NewInstance(argc, argv);
                   ~~~~~~^~~~~~~~~~~
/Users/antonverinov/.node-gyp/10.6.0/include/node/v8.h:3914:44: note: candidate function not viable: requires single argument 'context', but 2 arguments were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
/Users/antonverinov/.node-gyp/10.6.0/include/node/v8.h:3911:44: note: candidate function not viable: requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> NewInstance(
                                           ^
2 errors generated.
make: *** [Release/obj.target/node-cint64/src/cint64.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/antonverinov/.nvm/versions/node/v10.6.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Darwin 17.7.0
gyp ERR! command "/Users/antonverinov/.nvm/versions/node/v10.6.0/bin/node" "/Users/antonverinov/.nvm/versions/node/v10.6.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/antonverinov/workspace/node-cint64
gyp ERR! node -v v10.6.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-cint64@1.1.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-cint64@1.1.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by cactorium** [comment]
Thanks for the report! Yeah sorry, I honestly haven't used Node in any projects recently, and the bulk of this package was made quite a while ago. I think I have a fix, I'm gonna wait to see if the Travis CI passes for all the version I have listed. You can pull from master in the meantime if you wanna test it out first

**Comment by cactorium** [comment]
Please confirm if this is fixed on version 2.0.0

**Comment by zemlanin** [comment]
Yep, everything's great. Thanks 👍
